app/models/game.rb

```diff
@@ -3,10 +3,19 @@ class Game < ApplicationRecord
   belongs_to :player_1, class_name: 'User'
   belongs_to :player_2, class_name: 'User'
 
-  enum current_turn: ["challenger", "computer"]
+  #enum current_turn: ["challenger", "computer"]
+  enum current_turn: [:player_1, :player_2]
   serialize :player_1_board
   serialize :player_2_board
 
   validates :player_1_board, presence: true
   validates :player_2_board, presence: true
+
+  def swap_turn!
```

Comment:
I really like this swap turn! If we ever want to go back to the challenger computer, we could just have another process for the AI